### PR TITLE
Add qga

### PR DESCRIPTION
**Pull request by WuJun2016**
# links
- https://github.com/harvester/harvester/issues/1833

### CODE REVIEW

**Review by guangbochen — APPROVED** [review]
LGTM

File: docs/vm/create-vm.md [hunk 1]
```diff
@@ -62,6 +62,16 @@ config:
...
 
+### Installing the QEMU guest agent
+The QEMU guest agent is a daemon that runs on the virtual machine and passes information to the host about the virtual machine, users, file systems, and secondary networks.
+
+Qemu guest agent is installed by default when a VM is created.
+
+![](assets/qga.png)
+
+!!! note
+	If your OS is openSuse and the version is less than 15.3, please replace `qemu-guest-agent.service` with `qemu-ga.service`.
```
Comment (futuretea) [inline docs/vm/create-vm.md:73]: openSUSE

Comment (WuJun2016) [reply docs/vm/create-vm.md]: Done

**Review by futuretea — APPROVED** [review]
LGTM

**Pull request merged**